src/test/java/yandex/pages/YandexMailPage.java

```diff
@@ -0,0 +1,35 @@
+package yandex.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class YandexMailPage {
+
+    WebDriver driver;
+    WebDriverWait wait;
+    public static final By USERNAME_INPUT = By.name("login");
```

Comment:
please try to add more locators even for elements that will not be used in this test. Use all possible types of locators for WebDriver (xPath, name, classname, tagname, etc.)